azure-pipelines.yml

```diff
@@ -102,8 +102,47 @@ jobs:
                $(Slack.URL)
         condition: and(failed(), eq(variables['Build.SourceBranchName'], 'master'))
 
+  - job: Windows_signing
+    # Signing is a separate job so that we can make sure that we only sign on releases.
+    # Since the release check is run on Linux, we do not have access to that information
+    # in the regular Windows step.
+    dependsOn: [ "Windows", "Linux" ]
+    pool:
+      name: 'windows-pool'
+    condition: and(succeeded(), eq(dependencies.Linux.outputs['release.has_released'], 'true'))
```

Comment:
I’ve verified that signing works correctly without the `has_released` check so hopefully it will work properly during a release as well.